scripts/build_utils.sh

```diff
@@ -1334,15 +1334,14 @@ get_nr_build_jobs() {
 
 setup_rpm_build_deps() {
     $SUDO yum install -y yum-utils
-    if [ "$RELEASE" = 7 ]; then
+    if [ "$RELEASE" = 8 ]; then
         if [ "$ARCH" = x86_64 ]; then
-            $SUDO yum install -y centos-release-scl
+            $SUDO dnf install -y centos-release-scl
         elif [ "$ARCH" = arm64 ]; then
-            $SUDO yum install -y centos-release-scl-rh
-            $SUDO yum-config-manager --disable centos-sclo-rh
-            $SUDO yum-config-manager --enable centos-sclo-rh-testing
+            $SUDO dnf install -y centos-release-scl-rh
+            $SUDO dnf config-manager --disable centos-sclo-rh
+            $SUDO dnf config-manager --enable centos-sclo-rh-testing
```

Comment:
probably we don't need SCL for accessing GTS-11. see https://centos.pkgs.org/8-stream/centos-appstream-x86_64/gcc-toolset-11-build-11.0-0.el8.x86_64.rpm.html

what we need is but to enable the "AppStream" repo. and
```sh
dnf install gcc-toolset-11-build
```

Reply:
ok, thanks. i was being conservative by requiring gcc 10.2, but i can update https://github.com/ceph/ceph/pull/45133 with `BuildRequires: gcc-toolset-11-gcc-c++`

does that mean ubuntu should use_ppa 11 to match?

i'm not familiar with AppStream, and grep didn't find anything under ceph or ceph-build; is that repo expected to be enabled by default, or do i need something extra here?

Reply:
ah, i see that ceph.spec.in has `. /opt/rh/gcc-toolset-9/enable` for this, i'll make that unconditional and raise it to 11